<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QLabel" name="gamesLabel">
       <property name="text">
        <string>Ваши игры:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="gamesTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>301</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="middleLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Никнейм:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="friendNickLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="friendNameLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="friendDescLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="addFriendButton">
       <property name="text">
        <string>Добавить друга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gameNameLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Рейтинг:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gameRatingSpinBox">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Ссылка:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gameURLLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Описание:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="gameDescLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="addGameButton">
       <property name="text">
        <string>Добавить игру</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stateLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>301</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QLabel" name="friendsLabel">
       <property name="text">
        <string>Ваши друзья:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="friendsTable"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
